### ISSUE TITLE
conda skeleton broken?

### ISSUE THREAD

**Issue by AlbertDeFusco** [conda/conda-build#1368]
See https://github.com/conda/conda/issues/3406

I have the same problem with the latest versions

```
Current conda install:

               platform : osx-64
          conda version : 4.2.7
       conda is private : False
      conda-env version : 4.2.7
    conda-build version : 2.0.1
         python version : 3.5.2.final.0
       requests version : 2.10.0
       root environment : /Users/adefusco/Applications/anaconda3  (writable)
    default environment : /Users/adefusco/Applications/anaconda3
       envs directories : /Users/adefusco/Applications/anaconda3/envs
          package cache : /Users/adefusco/Applications/anaconda3/pkgs
           channel URLs : https://conda.anaconda.org/defusco/osx-64/
                          https://conda.anaconda.org/defusco/noarch/
                          https://repo.continuum.io/pkgs/free/osx-64/
                          https://repo.continuum.io/pkgs/free/noarch/
                          https://repo.continuum.io/pkgs/pro/osx-64/
                          https://repo.continuum.io/pkgs/pro/noarch/
            config file : /Users/adefusco/.condarc
           offline mode : False
```

**Comment by msarahan** [comment]
Sorry about this.  It did not show up during testing, and I'm unsure why.  I'm out on paternity leave for this week, but will fix this ASAP.

**Comment by sooneecheah** [comment]
https://github.com/conda/conda-build/pull/1384 <- I think this is the issue

**Comment by msarahan** [comment]
Fixed by #1384 #1387 #1399
